_posts/2015-11-07-Creme-de-la-Pods.markdown

```diff
@@ -0,0 +1,38 @@
+
+Keeping on top of the community's work is a challenge, with ~13,500 CocoaPods available and roughly 20 new pods every day. We work on CocoaPods to encourage  [contributions to OSS](https://github.com/CocoaPods/CocoaPods#project-goals). A by-product of this is that if you follow the whole stream of new CocoaPods, it can get hard to distinguish between "my first library" and "an awesome paradigm _swift_."
+
+I have taken a stab at solving this problem, by trying to select the best pods by their [Quality Index](https://guides.cocoapods.org/making/quality-indexes.html). The first time we run documentation parsing for a Pod, if that Pod's QI is over 70 then [CocoaDocs] (https://github.com/CocoaPods/cocoadocs-api/commit/3dd3ee32f253d485576fb49fa272945cd5d5462b) will send out a tweet on the [@CremeDeLaPods](https://twitter.com/cremedelapods) twitter feed, with the exact same formatting as [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed).
+
+<a class="twitter-follow-button" href="https://twitter.com/CremeDeLaPods">Follow @CremeDeLaPods</a>
+
+<!-- more -->
+
+So there should be no surprises here if you've seen tweets from [@CocoaPodsFeed](https://twitter.com/CocoaPodsFeed)
+
+<center>
+  <blockquote class="twitter-tweet" lang="en" data-cards="hidden"><p lang="en" dir="ltr">[Venice] CSP for Swift 2 (Linux ready) <a href="https://t.co/DdPyqojBjI">https://t.co/DdPyqojBjI</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/663133105809698821">November 7, 2015</a></blockquote>
+
+  <blockquote class="twitter-tweet" lang="en" data-cards="hidden"><p lang="fr" dir="ltr">[ControllerContainer by <a href="https://twitter.com/3lvis">@3lvis</a>] View Controller Containment for humans <a href="https://t.co/3WgHkIrUUa">https://t.co/3WgHkIrUUa</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/662624562778144770">November 6, 2015</a></blockquote>
+  </center>
+
+Every Pod also comes with an overview images too, making it easy to see some of the salient properties, we've been doing this for a few years.
+
+  <center>
+  <blockquote class="twitter-tweet" lang="en"><p lang="en" dir="ltr">[Money by <a href="https://twitter.com/danthorpe">@danthorpe</a>] Swift types for working with Money. <a href="https://t.co/jCdJlrZV9Z">https://t.co/jCdJlrZV9Z</a></p>&mdash; Creme De La Pods (@CremeDeLaPods) <a href="https://twitter.com/CremeDeLaPods/status/662573883728490496">November 6, 2015</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <a class="twitter-follow-button" href="https://twitter.com/CremeDeLaPods" data-size="large">Follow @CremeDeLaPods</a>
+</center>
```

Comment:
This link is already present a few lines above in the post, no need to display it twice.